scripts/rv_expression_association/debug_maf_distributions.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import hail as hl
+# from bokeh.io import show
+from bokeh.io.export import get_screenshot_as_png
+from cpg_utils.hail_batch import output_path, init_batch 
+
+init_batch()
+mt = hl.read_matrix_table('gs://cpg-tob-wgs-test/tob_wgs_vep/v1/vep105_GRCh38.mt')
```

Comment:
As discussed on Slack, this should point to `v7.mt` and use `dataset_path`, as the VEP annotated version is broken w.r.t. densification.

Reply:
i think this script is not needed anymore so will delete, but adopting the corrections for get_rv_outliers.py :)